install/install-sh

```diff
@@ -244,7 +244,20 @@ if [ "$sysname" == "Darwin" ]; then
         echo "Running pip install ${APP_ROOT}/sleuth/"
         pip install ${APP_ROOT}/sleuth/
     else
-        pip install -t $PREFIX/lib/python ${APP_ROOT}/sleuth/
+	cd ${APP_ROOT}/sleuth
+	echo "Running python setup.py bdist --format=gztar"
+	python setup.py bdist --format=gztar
+        if [ $retval -ne "0" ]; then
+            echo "error: could not run python setup.py for sleuth"
+            exit 1
+        fi
+        SLEUTHFILE="${APP_ROOT}/sleuth/dist/sleuth-0.1.macosx-10.12-intel.tar.gz"
```

Comment:
If the version for Sleuth is changed, will this line break?

Reply:
I just committed a fix to grab Sleuth version info and MAC OSX version for tarball filename.